pkgs/development/python-modules/python-snappy/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, pythonPackages, fetchPypi, snappy }:
```

Comment:
Don't use `pythonPackages` within `python-packages.nix`. `python` is already an available attribute so use that as parameter.

Reply:
Done